test/osm_files/Denver.osm

```diff
@@ -1,41 +1,41 @@
 !- NOTE: Auto-generated from /test/osw_files/Denver.osw
 
 OS:Version,
-  {58417475-5018-41b0-8198-338d903fcf41}, !- Handle
-  2.9.0;                                  !- Version Identifier
+  {5472002e-fd4b-4888-b9e1-c149780e1171}, !- Handle
+  2.9.1;                                  !- Version Identifier
```

Comment:
Looks like you regenerated with os 2.9.1. I would stick with 2.9.0.

Reply:
@joseph-robertson Thanks, addressed in SHA 0569578.